inspirehep/modules/records/receivers.py

```diff
@@ -54,6 +54,7 @@
     get_push_access_tokens,
     get_orcids_for_push,
 )
+from elasticsearch import NotFoundError
```

Comment:
In python there is a convention for import statements order: 
```
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.
```
Sooo, you should move this guy upper :stuck_out_tongue: 

https://www.python.org/dev/peps/pep-0008/#imports